code/datums/craft/recipes/guns.dm

```diff
@@ -185,7 +185,7 @@
 	name = "\"STS\" battle rifle"
 	result = /obj/item/gun/projectile/automatic/sts/rifle
 	steps = list(
-		list(/obj/item/gun/projectile/automatic/sts/lrifle, 1, "time" = 60),
+		list(/obj/item/gun/projectile/automatic/sts, 1, "time" = 60),
```

Comment:
```suggestion
		list(/obj/item/gun/projectile/automatic/sts/lrifle, 1, "time" = 60),
```

Reply:
All instances of "lrifle" are deleted, why are you bringing them back? Because you can accidentally make a battle rifle with another battle rifle or the PARA since it just asks for a parent STS?